code/API_definitions/edge-discovery-service.yaml

```diff
@@ -11,14 +11,198 @@ info:
   description: |
     # Introduction
     ---
-    Use the Edge Discovery API for
-    discovery of the most optimal edge cloud zone for a specific user as well as specific application instance the user is trying to connect to.
+    The Edge Discovery API is designed to help users discover the most optimal
+    edge cloud zones for their applications. This API allows users to find the
+    best regions and zones for their edge cloud applications, discover optimal
```

Comment:
One generic recommendation would be to use "optimal" instead of "best" as we discussed this in past meetings as that may be hard to determine best edge for various reasons.